VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddCommentFragment.java

```diff
@@ -153,4 +154,18 @@ public void onException(Exception ex) {
         createCommentTask.setProgressDialog(createCommentProgressBar);
         createCommentTask.execute();
     }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+
+        if (getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE) {
```

Comment:
This isn't immediately obvious to me. Why are  we only clearing the field on landscape mode? Wouldn't it be whenever the orientation changes?